scripts/build_diagrams.py

```
@@ -0,0 +1,209 @@
+                    tspan.set("font-weight", "700")
+    for marker in root.findall(f".//{{{SVG_NS}}}marker"):
+        if "pointEnd__" in marker.get("id", ""):
+            for attr, value in ARROW_MARKER_ATTRS.items():
+                marker.set(attr, value)
+    return ET.tostring(root, encoding="unicode")
+
+
+def render_theme(source: Path, theme: str, config: dict[str, str], tmp_dir: Path, digest: str) -> tuple[Path, str]:
+    """Render one source/theme pair and return the destination with stamped SVG."""
+    tmp = tmp_dir / f"{source.stem}-{theme}.svg"
+    subprocess.run(
```

Ruff (0.15.15)

[error] `subprocess` call: check for execution of untrusted input

(S603)